packages/loader/container-definitions/src/runtime.ts

```diff
@@ -121,7 +121,9 @@ export interface IContainerContext extends IDisposable {
     readonly quorum: IQuorum;
     readonly audience: IAudience | undefined;
     readonly loader: ILoader;
+    /** @deprecated - use taggedLogger instead */
```

Comment:
can you add more documentation here. it's quite strange to have two properties of the same type